lib/doorkeeper/orm/active_record.rb

```diff
@@ -12,6 +12,27 @@ def self.initialize_models!
           end
         end
       end
+
+      def self.initialize_application_owner!
```

Comment:
> I remember some people want to override it when they implement new ORM adapter

Do you remember the specific cases where this change makes it better?


> this is an option, so keep separate between `initialize_models!`

It was already its own method in the config file. Why is it better to "unroll" it in each ORM implementation?